src/Multiselect.php

```diff
@@ -357,4 +357,26 @@ public function setOptionsFromModels(Collection $models, $resourceClass)
         });
         $this->options($options);
     }
+
+    /**
+     * Sets variables which will be used for displaying index value
+     *
+     * @param  string $delimiter
+     * @param  int $limit
+     * @return \OptimistDigital\MultiselectField\Multiselect
+     */
+    public function displayIndexUsing($delimiter = ', ', $limit = 40)
+    {
+        return $this->withMeta(['delimiter' => $delimiter, 'displayAtOnce' => $limit]);
```

Comment:
Couldn't decide between having one or two functions for setting the values. 
Alternatively, we can do `delimiter()` and `maxValuesOnIndex()`.